include/world_builder/utilities.h

```diff
@@ -249,6 +249,57 @@ namespace WorldBuilder
         std::vector<double> m_a, m_b, m_c, m_y;
     };
 
+    /**
+     * A struct that is used to hold the return values of the function
+     * distance_point_from_curved_planes(). See there for a documentation
+     * of the meaning of the member variables.
+     */
+    struct PointDistanceFromCurvedPlanes
+    {
+      /**
+       * The shortest distance between point and plane.
+       */
+      double distance_from_plane;
+
+      /**
+       * The distance between edge of the plane and
+       * the point if the point is projected onto the curved plane.
```

Comment:
I do not understand what you mean with edge of the plane here, so I am not sure whether this description is correct. This should be something like: 
```
The distance between the the start of the first segment (usually at the surface) to point a point on the (curved) plane to the provided point following the (curved) plane.